brambling/templates/brambling/event/organizer/update.html

```diff
@@ -1,9 +1,295 @@
+{% load floppyforms %}
```

Comment:
There should probably be some integration tests that catch these sorts of things someday...